civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -10,33 +10,88 @@ use Drupal\civihr_employee_portal\Helpers\HelperClass;
  */
 function civihr_employee_portal_css_alter(&$css) {
 
-    // Remove defaults system messages css file.
-    unset($css[drupal_get_path('module', 'system') . '/system.messages.css']);
+  // Remove defaults system messages css file.
+  unset($css[drupal_get_path('module', 'system') . '/system.messages.css']);
+
+  // Removing unwanted css files from civicrm pages.
+  if (civicrm_invoke()) {
```

Comment:
I'm saying that it seems to me that it does quite a lot of things to be used just as a simply utility function. Isn't there a better way? Or maybe i'm getting it wrong and it's totally fine to use it? What was your reasoning to use that function specifically?